### PR TITLE
Add target to expose the script, so it can be added to deb

### PR DESCRIPTION
**Pull request by costinm**

### CODE REVIEW

File: docker/BUILD [hunk 1]
```diff
@@ -48,6 +48,14 @@ pkg_tar(
     tags = ["manual"],
 )
 
+filegroup(
+    name = "prepare_proxy",
+    srcs = [
+        "prepare_proxy.sh",
```
Comment (kyessenov) [inline docker/BUILD]: change prepare_proxy.sh below to this target?

**Comment by kyessenov** [comment]
do you still need this PR?

**Comment by costinm** [comment]
Yes still needed, have to merge and get the tests passed (sorry got distracted by other issues).

**Comment by costinm** [comment]
Will create a new PR after merge

**Pull request closed without merging**